lib/literature/live/blog_live.ex

```diff
@@ -204,6 +204,32 @@ defmodule Literature.BlogLive do
     |> assign(:authors, list_authors(socket))
   end
 
+  defp apply_action(socket, :show_author, slug, _params) do
+    publication = Literature.get_publication!(slug: slug)
+    meta_tags = get_meta_tags_from_view_module(socket, :show_author, socket.assigns.author)
+
+    socket =
+      case meta_tags do
+        %{title: _} -> assign_meta_tags(socket, meta_tags)
+        %{} -> socket
+      end
+
+    assign(socket, :publication, publication || %{name: nil})
+  end
+
+  defp apply_action(socket, :show_tag, slug, _params) do
```

Comment:
I think we can just add a guard `when action in [:show_tag, :show_author]` so we only have 1 addtional `apply_action` function